api-server/src/server/boot/authentication.js

```diff
@@ -188,3 +190,41 @@ function createGetPasswordlessAuth(app) {
     );
   };
 }
+
+function mobileLogin(app) {
+  const {
+    models: { User }
+  } = app;
+  return async function getPasswordlessAuth(req, res, next) {
+    const {
+      headers: { token: accessToken }
+    } = req;
+
+    const userInfo = await axios({
+      method: 'GET',
+      url: `https://${process.env.AUTH0_DOMAIN}/userinfo`,
+      headers: { Authorization: `Bearer ${accessToken}` }
+    });
```

Comment:
Axios is a particularly heavy binary. I do not see it being used other than this call. This could be done in vanilla JavaScript instead.

Reply:
When I tried using fetch instead of Axios it said that fetch was not defined. Normally you use Node fetch or Axios to do this.

Reply:
Correct. 

It would be best if you had https://www.npmjs.com/package/node-fetch (Node.js v16 and below), which is much lighter than Axios. 

There is also a native fetch API in Node.js v18 (we are moving to this very shortly). See: https://nodejs.org/de/blog/announcements/v18-release-announce

Reply:
Yes, Axios was already a dependency.

Reply:
Yep. Thinking about it, there are two paths forwards that seem fine to me.

1. use node-fetch here and migrate donation to node-fetch separately
2. use axios here and migrate both to node-fetch in another PR

Maybe 1. is best because it limits the scope. Also, the api already has `node-fetch` as a transitive dependency so having it explicitly will not change anything about the installation.

Reply:
I'm in favour of 1 as it's better to use node-fetch now itself and well not forget to change from Axios to node-fetch in future